src/benchmarks/micro/runtime/Math/Functions/Double/Exp2.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using BenchmarkDotNet.Attributes;
+
+namespace System.MathBenchmarks
+{
+    public partial class Double
+    {
+        // Tests double.Exp2(double) over 5000 iterations for the domain -1, +1
+
+        private const double exp2Delta = 0.0004;
+        private const double exp2ExpectedResult = 5410.8564379907702;
```

Comment:
This value is different than Exp2M1.cs

Reply:
I see the confusion.

This is not "Apple M1", but rather a function `Exp2M1` which represents `Exp2 Minus One` (`Exp(2) - 1`) but computed more accurately.